modules/apps/batch-engine/batch-engine-api/src/main/java/com/liferay/batch/engine/jaxrs/uri/BatchEngineUriInfo.java

```diff
@@ -150,7 +150,13 @@ public Builder taskItemDelegateName(String taskItemDelegateName) {
 
 	private BatchEngineUriInfo(Builder builder) {
 		_pathParameters = builder._pathParameters;
+
 		_queryParameters = builder._queryParameters;
+
+		if (_queryParameters.containsKey("siteId")) {
```

Comment:
I am not sure about including this if clause here, as we are considering the `scopeKey` parameter is going to be always set with the `siteId` value.

What do you think about considering always in the Batch layer `siteId` as `siteId`, and then updating the implementation of the `read` method of the VulcanBatch interface to change the way the `scopeKey` value is read from the parameters in the `ObjectEntryResourceImpl`, considering both options?.

Something like:

```
@Override
public Page<ObjectEntry> read(
		Filter filter, Pagination pagination, Sort[] sorts,
		Map<String, Serializable> parameters, String search)
	throws Exception {

	_loadObjectDefinition(parameters);

	ObjectScopeProvider objectScopeProvider =
		_objectScopeProviderRegistry.getObjectScopeProvider(
			_objectDefinition.getScope());

	if (objectScopeProvider.isGroupAware()) {
		return getScopeScopeKeyPage(
			_getScopeKey(parameters),
			Boolean.parseBoolean((String)parameters.get("flatten")), search,
			null, filter, pagination, sorts);
	}

	[...]

}
```
being:
```
private String _getScopeKey(Map<String, Serializable> parameters) {
	if (parameters.containsKey("scopeKey")) {
		return String.valueOf(parameters.get("scopeKey"));
	}

	if (parameters.containsKey("siteId")) {
		return String.valueOf(parameters.get("siteId"));
	}

	return null;
}

```